docs/source/guide/tasks.md

```diff
@@ -308,11 +420,11 @@ If you want to import entire plain text files without each line becoming a new l
 
 ### Import HTML data
 
-You can import HyperText data in HTML-formatted files and annotate them in Label Studio. When you directly import HTML files, the content is minified the content is minified by compressing the text, removing whitespace and other nonfunctional data in the HTML code. Annotations that you create are applied to the minified version of the HTML. 
+You can import `HyperText` data in HTML-formatted files and annotate them in Label Studio. When you directly import HTML files, the content is minified the content is minified by compressing the text, removing whitespace and other nonfunctional data in the HTML code. Annotations that you create are applied to the minified version of the HTML. 
```

Comment:
here we need to explain what happens with JS code inside HTML docs

Reply:
also `the content is minified` is duplicated here

Reply:
JS in HTML is completely out of the topic of this page and topic of this PR. The best place is HyperText tag doc, I think

Reply:
will be done in DEV-2881